berkeley-function-call-leaderboard/bfcl_eval/.env.example

```diff
@@ -27,6 +27,9 @@ AWS_SECRET_ACCESS_KEY=
 DATABRICKS_API_KEY=
 DATABRICKS_AZURE_ENDPOINT_URL=
 
+# We use the API Key from Alipay to inference Bailing (Ling) models (see https://zxb.alipay.com/llm/landing)
+LING_API_KEY = sk-XXXXXX
```

Comment:
We shouldn't have space around the equal sign. This will include the space as part of the key or value, eg, treat `LING_API_KEY ` (with a trailing space) as the key.